src/test/scala/io/citrine/lolo/trees/regression/RegressionTreeTest.scala

```diff
@@ -148,6 +147,25 @@ class RegressionTreeTest {
     val oos = new ObjectOutputStream(new FileOutputStream(tmpFile))
     oos.writeObject(DT)
   }
+
+  /**
+    * Test a really short tree to make sure the linear model feature importance gets carried through
+    */
+  def testShortTreeWithLinearLeaf(): Unit = {
+     val trainingData = TestUtils.generateTrainingData(1024, 12, noise = 0.1, function = Friedman.friedmanSilverman)
+
```

Comment:
It might also be good to do another test with a tree with a single leaf node, so that it's deterministic (can you turn off bagging for your trees?).  This test is good, but since it's not deterministic, it might good to also have a deterministic one.